### PR TITLE
Follow up to merge request 1062 and 1068

### PR DESCRIPTION
**Pull request by Pierre-Sassoulas**
## Description

Just two things forgotten during review.

## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :hammer: Refactoring   |

## Related Issue

#1062

### CODE REVIEW

**Review by cdce8p — CHANGES_REQUESTED** [review]
It should be `sys.version_info >= (3, 10)`. That's the preferred version for type checkers.

**Comment by Pierre-Sassoulas** [comment]
I think this constant is misleading as in pylint it's named PY310_PLUS instead but it's the same one: https://github.com/PyCQA/astroid/blob/master/astroid/const.py#L7 Do you want to change them all to PY3X_PLUS ?

**Comment by cdce8p** [comment]
> Do you want to change them all to PY3X_PLUS ?

IMO we should remove them completely.
https://mypy.readthedocs.io/en/stable/common_issues.html#python-version-and-system-platform-checks

**Comment by cdce8p** [comment]
Regarding the 930+ errors: I think that's because of `PY310`

**Comment by Pierre-Sassoulas** [comment]
> IMO we should remove them completely.

It's not used only for mypy though, they are used in a lot of places.

**Comment by cdce8p** [comment]
> It's not used only for mypy though, they are used in a lot of places.

Let's change the existing ones to `PYXX_PLUS` then and keep `sys.version_info` where it's required for type checking.

**Comment by Pierre-Sassoulas** [comment]
I think that everything is ready after that one, but I'm going to release 2.6.0 tomorrow (and after checking in pylint first), I've coded too much today 😄

**Comment by cdce8p** [comment]
Can you add the additional typing dependencies to `requirements_test_brain.txt` / `requirement_test.txt`?

File: requirements_test.txt [hunk 1]
```diff
@@ -5,3 +5,5 @@ coverage~=5.5
 pre-commit~=2.13
 pytest-cov~=2.11
 tbump~=6.3.2
+types-typed-ast
```
Comment (cdce8p) [inline requirements_test.txt:8]:
```suggestion
types-typed-ast; implementation_name=="cpython" and python_version<"3.8"
```
Must sure if it's necessary as this is just a types package. Shouldn't harm though

**Review by cdce8p — APPROVED** [review]
Looks good overall. Thanks for doing it 🚀

**Pull request merged**